### PR TITLE
PMM-5777 Inventory add external exporter.

### PR DESCRIPTION
**Pull request by BupycHuk**

### CODE REVIEW

File: commands/inventory/add_agent_external_exporter.go [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+
+var addAgentExternalExporterResultT = commands.ParseTemplate(`
+External Exporter added.
+Agent ID              : {{ .Agent.AgentID }}
+Runs on Node ID       : {{ .Agent.RunsOnNodeID }}
+Service ID            : {{ .Agent.ServiceID }}
+Username              : {{ .Agent.Username }}
+Scheme                : {{ .Agent.Scheme }}
+Metric Path           : {{ .Agent.MetricPath }}
+Listen port           : {{ .Agent.ListenPort }}
```
Comment (percona-csalguero) [inline commands/inventory/add_agent_external_exporter.go:33]: Sorry for this picky comment but Path, Node, ID words are capitalized but port and lables don't

**Review by percona-csalguero — APPROVED** [review]

**Review by artemgavrilov — APPROVED** [review]

File: commands/inventory/add_agent_external_exporter.go [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+
+type addAgentExternalExporterCommand struct {
+	RunsOnNodeID string
+	ServiceID    string
+	Username     string
+	Password     string
+	CustomLabels string
+	Scheme       string
+	MetricsPath  string
+	ListenPort   int64
```
Comment (askomorokhov) [inline commands/inventory/add_agent_external_exporter.go:57]: Can we use uint16 here?

**Review by askomorokhov — APPROVED** [review]
Looks good to me, with a question.

File: commands/inventory/add_agent_external_exporter.go (hunk 2 above)
Comment (BupycHuk) [reply commands/inventory/add_agent_external_exporter.go:57]: Yes, we can, but we use int64 for ports in other commands for inventory API

**Pull request merged**